realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -789,6 +789,23 @@ public Table pivot(long stringCol, long intCol, PivotType pivotType){
         return result;
    }
 
+    /**
+     * Returns a {@link TableView} with distinct set of objects of a specific class.
+     * As a Realm is unordered, it is undefined which objects are returned in case of multiple
+     * occurrences.
+     *
+     * @param columnIndex the column index.
+     * @return a non-null {@link TableView} containing the distinct objects.
+     * @throws IllegalArgumentException if the type of the column is unsupported.
+     * UnsupportedOperationException if a column is not indexed.
+     */
+    public TableView getDistinctView(long columnIndex) {
```

Comment:
why call this `getDistinctView`? Isn't `distinct` enough since this won't create any new `TableView`?